src/v/cluster/controller_backend.cc

```diff
@@ -382,9 +385,11 @@ ss::future<> controller_backend::start() {
                     });
               });
         }
-        start_topics_reconciliation_loop();
-        _housekeeping_timer.set_callback([this] { housekeeping(); });
-        _housekeeping_timer.arm(_housekeeping_timer_interval);
+
+        // unblock reconciliation fibers
+        _reconciliation_sem.signal(1024);
```

Comment:
why do we need to defer depositing the units ? don't we star the reconciliation loop in the next call ?

Reply:
nit: if you can replace magic number with some (self) descriptive variable.